cfme/infrastructure/virtual_machines.py

```diff
@@ -222,15 +222,24 @@ def on_details(self, force=False, is_vm=True):
             else:
                 return False
 
-    def _find_quadicon(self, is_vm=True, do_not_navigate=False, mark=False, refresh=True):
+    def _find_quadicon(
+            self, is_vm=True, do_not_navigate=False, mark=False, refresh=True, any=False):
         """Find and return a quadicon belonging to a specific vm
 
+        Args:
+            any: Whether to look for it anywhere (root of the tree)
+
         Returns: :py:class:`cfme.web_ui.Quadicon` instance
         Raises: VmNotFound
         """
         quadicon = Quadicon(self.name, "vm")
         if not do_not_navigate:
-            if is_vm:
+            if any:
```

Comment:
I was playing with the unification using the base class but that would probably be something I could spend my time without context switching to extract and unify the common things. I even have a branch where I tried that :)